src/codegen/riscv64/macro-assembler-riscv64.cc

```diff
@@ -477,31 +477,30 @@ void TurboAssembler::Mul(Register rd, Register rs, const Operand& rt) {
   }
 }
 
+// MIPS-style mulh: top 32-bits of a 64-bit product.
```

Comment:
What do you think about renaming these methods instead of just adding the comments to make it more clear? Something like `Mul32Upper`?

Reply:
I think we want to just get rid of them in the long run.  We'll get better code generation in the long run if we do this during instruction selection rather than TMA.  For the short term I don't really care either way, this was just easier.  If you want me to  rename them I'm happy to do so.

Reply:
Good point about removing them long term.